web_src/js/features/comp/EditorUpload.ts

```diff
@@ -109,13 +111,13 @@ async function handleUploadFiles(editor, dropzoneEl, files, e) {
   }
 }
 
-export function removeAttachmentLinksFromMarkdown(text, fileUuid) {
+export function removeAttachmentLinksFromMarkdown(text: string, fileUuid: string) {
   text = text.replace(new RegExp(`!?\\[([^\\]]+)\\]\\(/?attachments/${fileUuid}\\)`, 'g'), '');
   text = text.replace(new RegExp(`<img[^>]+src="/?attachments/${fileUuid}"[^>]*>`, 'g'), '');
   return text;
 }
 
-function handleClipboardText(textarea, e, {text, isShiftDown}) {
+function handleClipboardText(textarea: HTMLTextAreaElement, e: ClipboardEvent, text: string, isShiftDown: boolean) {
```

Comment:
It was required by you to do so

![image](https://github.com/user-attachments/assets/64a812b4-cf39-45a0-8da2-c5c435d9fe34)

Reply:
That was before TS, and I think I came to the conclusion that bare args are more ergonomic to write types for.

Reply:
My case, there is no "bool" trick:

```
isShiftDown = ...;
...
handleClipboardText(textarea, e, text, isShiftDown);
```

----

You case: 

```
await attachTribute(this.easyMDE.codemirror.getInputField(), true, true);
```

----

TBH, I do not think writing `foo(..., true, true)` is readable: what are these `true` values for?

Reply:
Leave this as-is? My main point about `isShiftDown` was to separate required from optional args, but that really only pays off if there is more than one option.